CMakeLists.txt

```diff
@@ -810,14 +822,9 @@ foreach(sourcefile ${C_TEST_EXES})
     add_test(NAME ${exename} COMMAND ${exename}${ARTIFACT_SUFFIX})
     add_dependencies(check ${exename}${ARTIFACT_SUFFIX})
 endforeach(sourcefile ${C_TEST_EXES})
-add_subdirectory(tools)
+endif()
 
-# Installation and packaging for Linux
-if(NOT WIN32)
```

Comment:
> any ideas?

@chfast I don't see the point of making it conditional. How user should use libraries if there will be no way to install it? Though it doesn't hurt if it will be under some condition.

> How about SNAPPY_INSTALL_ON_WINDOWS?

Sounds good.